crates/symbolicator/src/services/cacher.rs

```diff
@@ -308,26 +308,39 @@ impl<T: CacheItemRequest> Cacher<T> {
         let dup_file = temp_file.reopen()?;
         let mut temp_fd = tokio::fs::File::from_std(dup_file);
 
-        // TODO: consider cache expiry!
         let shared_cache_hit = self
             .shared_cache_service
             .fetch(&shared_cache_key, &mut temp_fd)
             .await;
 
-        let status = if !shared_cache_hit {
-            let status = request.compute(temp_file.path()).await?;
-            status.write(&mut temp_fd).await?;
-            Some(status)
-        } else {
-            None
+        let status = match shared_cache_hit {
+            true => {
```

Comment:
match on true/false is super weird, I wonder why clippy does not complain, lol

Reply:
heh, just style i think, i generally prefer match over if...else.  doubly so when assigning the result of the expression.